config/environments/development.rb

```diff
@@ -42,4 +42,10 @@
   }
 
   config.action_controller.action_on_unpermitted_parameters = :raise
+
+  config.after_initialize do
```

Comment:
It's in development.rb, so I don't think it's loaded in prod unless we've got some super weirdo rails setup.